plugins/org.jkiss.dbeaver.model.sql/src/org/jkiss/dbeaver/model/sql/completion/SQLCompletionAnalyzer.java

```diff
@@ -1317,7 +1317,7 @@ public void visit(@Nullable CreateView createView) {
                             if (aliases.contains(s) || sqlDialect.getKeywordType(s) != null) {
                                 return true;
                             }
-                            return extractTableName(s, false) != null;
+                            return extractTableNames(s, false).isEmpty();
```

Comment:
We want to make sure that there are results, don't we?

```suggestion
                            return !extractTableNames(s, false).isEmpty();
```